File: library/src/main/java/ru/terrakok/cicerone/result/ResultListener.java
/*
 * Created by Konstantin Tskhovrebov (aka @terrakok)
 */

package ru.terrakok.cicerone.result;

public interface ResultListener {

    /**
     * Received result from screen.
     *
     * @param resultData
     */
    void onResult(Object resultData);
}

File: library/src/main/java/ru/terrakok/cicerone/result/ResultListener.kt
package ru.terrakok.cicerone.result

/**
 * @author Konstantin Tskhovrebov (aka terrakok) on 04.07.17.
 */
interface ResultListener {

    /**
     * Received result from screen.
     *
     * @param resultData
     */
    fun onResult(resultData: Any)
}
